client-course-schedulizer/src/utilities/helpers/caseFunctions.ts

```diff
@@ -27,12 +27,40 @@ const thursReg = RegExp("[Tt][Hh]|[Rr]");
 const friReg = RegExp("[Ff]");
 const satReg = RegExp("[Ss](?![Uu])");
 
-export const startTimeCallback = (value: string, { firstMeeting }: CaseCallbackParams) => {
-  firstMeeting.startTime = startTimeCase(value);
+export const startTimeCallback = (value: string, { meetings }: CaseCallbackParams) => {
+  const startTimes = value.split("\n");
+  let i = 0;
+  startTimes.forEach((startTime) => {
+    if (meetings.length <= i) {
```

Comment:
The `forEach` function has a second parameter for the index, so there's no need to keep track of it manually.

`startTimes.forEach((startTime, i) => { ... })`

Reply:
This is really good to know, thanks! cac67a9